x/concentrated-liquidity/incentives.go

```diff
@@ -592,8 +645,20 @@ func (k Keeper) createIncentive(ctx sdk.Context, poolId uint64, sender sdk.AccAd
 		MinUptime:        minUptime,
 	}
 
+	// Get all incentive records for uptime
+	existingRecordsForUptime, err := k.GetAllIncentiveRecordsForUptime(ctx, poolId, minUptime)
+	if err != nil {
+		return types.IncentiveRecord{}, err
+	}
+
+	// Fixed gas consumption per swap to prevent spam
+	ctx.GasMeter().ConsumeGas(uint64(types.BaseGasFeeForNewIncentive*len(existingRecordsForUptime)), "balancer swap computation")
+
```

Comment:
On the first record, will this consume zero gas *total* or zero gas *extra*?

Reply:
Zero gas extra, as `ConsumeGas` is purely additive to total consumed gas